RELEASES.md

```diff
@@ -0,0 +1,90 @@
+# Releases
+
+- [Releases](#releases)
+  - [Semantic Versioning](#semantic-versioning)
+  - [Release Cycle](#release-cycle)
+  - [Stable Release Policy](#stable-release-policy)
+  - [Version Matrix](#version-matrix)
+    - [Backwards Compatibility](#backwards-compatibility)
+
+## Semantic Versioning 
+
+Interchain Security (ICS) follows [semantic versioning](https://semver.org), but with the following deviations (similar to [IBC-Go](https://github.com/cosmos/ibc-go/blob/main/RELEASES.md)):
+
+- A library API breaking change will result in an increase of the MAJOR version number (X.y.z | x > 0). 
+- A state-machine breaking change (change requiring coordinated upgrade and/or state migration for the consumer, the provider, or both) will result in an increase of the MINOR version number (x.Y.z | x > 0).
```

Comment:
It might be nice to have short examples (maybe links to existing PRs) of what constitutes "state-machine breaking change," "library API breaking change," and "node API breaking change".

Reply:
I added some more context on breaking changes. I don't want to refer to existing PRs as this may make people focus too much on those PRs, instead of focusing on the actual definitions.

Reply:
Also see https://github.com/cosmos/interchain-security/pull/1429/commits/ef0b3cc78e588e9ca61efaf3bbfe1fe8188476e6

Reply:
Amazing. Thank you!